README.md

```diff
@@ -669,6 +671,37 @@ Use this script, and ensure the CHANGELOG.md PR is merged, and the repo is
 tagged and released, before you run
 [release_collection.py](#release_collectionpy).
 
+# role-merge-prs.sh
+
+Used in conjunction with `manage-role-repos.sh`.  This will list the PRs for a
+role and prompt for merging one of them.  The list will include the checks and
+tests status for the PR.  You can use `v NUM` to view the PR in the UI and see
+the tests and checks details.  Enter `NUM`, where `NUM` is the PR number, to
+merge the PR and delete the branch.  This uses `gh pr merge NUM -r -d` to merge
+the PR.  If merging requires admin rights to override the checks, and you are an
+admin, you can use `a NUM` to merge.  This uses `gh pr merge NUM -r -d --admin`.
+By default, the list will only include PRs authored by you.  Use
+`AUTHOR=default` to view all PRs not authored by the `systemroller` account.
+Use `AUTHOR=username` to view those PRs authored by the given username.
+
+Examples:
+
+`./manage-role-repos.sh ./role-merge-prs.sh` - View all PRs authored by myself
+
+`AUTHOR=default ./manage-role-repos.sh ./role-merge-prs.sh` - View all PRs
+authored not by the `systemroller` account.
+
+`AUTHOR=richm ./manage-role-repos.sh ./role-merge-prs.sh` - View all PRs
+authored by richm.
+
+* `AUTHOR` - default is `@me` - look for PRs authored by this username.  Use
+  `default` to list all PRs except `systemroller` PRs.
```

Comment:
`default` is to avoid viewing the PR that runs weekly CI right?

Reply:
> `default` is to avoid viewing the PR that runs weekly CI right?

Right - but using `--search draft:false` is better since those PRs are marked draft